### DISCUSSION TITLE
Postgres Connection pool min value

### DISCUSSION THREAD

**Discussion by netum-villekuhlman** [sequelize/sequelize#13947] [Help & Questions] [answered]
Does Sequelize still support connection pool min value for Postgres?
For node-postgres library it was seemly removed (https://github.com/brianc/node-postgres/issues/2011)
 
At least when I define the Sequelize instance:

```
pool: {
    max: 25,
    min: 10,
    acquire: 30000,
    idle: 10000
  }
```

... and list active connections in the DB: 

`SELECT * FROM pg_stat_activity;`

... I can find only total 5 connections reserved for Sequelize client

**Comment by ephys** [comment, marked as answer]
We use `sequelize-pool` instead of `pg-pool` and from looking at the code, sequelize does pass the `min` option to `sequelize-pool` which supports it.

**Reply by netum-villekuhlman** [reply]
I debugged more on this and it seems that the `min` option works up to value `5`.
E.g. using min value 3 returns correctly 3 results from `pg_stat_activity`

But when using values above 5 e.g. 10 or 15, the query returns always 5 results from `pg_stat_activity`

What could cause the pool be limited to 5 as Postgres `max_connetion` is set to `100`?

**Reply by ephys** [reply]
I'm not sure, but I think this is the relevant issue https://github.com/sequelize/sequelize-pool/issues/22

**Reply by netum-villekuhlman** [reply]
Thanks, it is most likely related to this issue.
